syncnotification.go

```diff
@@ -34,99 +30,397 @@ func (lw *LibWallet) spvSyncNotificationCallbacks(loadedWallet *wallet.Wallet) *
 		RescanProgress:               generalNotifications.RescanProgress,
 		RescanFinished:               generalNotifications.RescanFinished,
 		PeerDisconnected: func(peerCount int32, addr string) {
-			for _, syncResponse := range lw.syncProgressListeners {
-				syncResponse.OnPeerDisconnected(peerCount)
-			}
+			lw.handlePeerCountUpdate(peerCount)
```

Comment:
Please re-order the `PeerDisconnected` and `PeerConnected` callbacks to make them easier to scan. Should be the first set of callbacks listed in this `*spv.Notifications` object.

This `spvSyncNotificationCallbacks` calls `lw.generalSyncNotificationCallbacks()` hence that function (`lw.generalSyncNotificationCallbacks`) is immediately below `spvSyncNotificationCallbacks`.

And immediately below `lw.generalSyncNotificationCallbacks` is `lw.handlePeerCountUpdate`, which is why I'm recommending that the `PeerDisconnected` and `PeerConnected` callbacks go to the top since they are the ones that call `lw.handlePeerCountUpdate()`

Arranging functions this way makes it easier to guess where a function could be in a long file just by looking at the hierarchy of function calls.